### PR TITLE
Io tweaks

### PR DESCRIPTION
**Pull request by sklum**
@jklontz: Minor stuff, but please review.

### CODE REVIEW

File: openbr/plugins/io/download.cpp [hunk 1]
```diff
@@ -57,7 +57,11 @@ class DownloadTransform : public UntrainableMetaTransform
         if (!url.contains("://"))
             url = "file://" + url;
         dst.file.set("URL", url);
-        if (url.startsWith("file://"))
+
+        QRegularExpression regExp("file:///[A-Z]:/");
```
Comment (jklontz) [inline openbr/plugins/io/download.cpp]: make this static const?

File: openbr/plugins/gallery/binary.cpp [hunk 2]
```diff
@@ -284,7 +286,7 @@ class utGallery : public BinaryGallery
             qFatal("Expected 16-byte ImageID, got: %d bytes.", imageID.size());
 
         const int32_t algorithmID = (t.isEmpty() || t.file.fte) ? 0 : t.file.get<int32_t>("AlgorithmID");
-        const QByteArray url = t.file.get<QString>("URL", t.file.name).toLatin1();
+        const QByteArray url = QUrl::fromUserInput(t.file.get<QString>("URL", t.file.name)).toEncoded();
```
Comment (jklontz) [inline openbr/plugins/gallery/binary.cpp]: Can you add a comment here explaining?

**Comment by jklontz** [comment]
looks good in general, but these are the kind of small tweaks that need to come with comments so we remember why we changed them.

**Comment by sklum** [comment]
@jklontz see updates.

**Comment by jklontz** [comment]
lgtm

**Pull request merged**